aspnetcore/performance/caching/memory.md

```diff
@@ -150,39 +143,40 @@ The size of the cache entry can be set by <xref:Microsoft.Extensions.Caching.Mem
 * Items with the earliest absolute expiration.
 * Items with the earliest sliding expiration.
 
-Pinned items with priority <xref:Microsoft.Extensions.Caching.Memory.CacheItemPriority.NeverRemove> are never removed. The following code removes a cache item and calls `Compact`:
+Pinned items with priority <xref:Microsoft.Extensions.Caching.Memory.CacheItemPriority.NeverRemove> are never removed. The following code removes a cache item and calls `Compact` to remove 25% of cached entries:
```

Comment:
Is my never link a distraction or useful? You have a similar link below. Never used to be a suggestion and in some caches it's a suggestion. When memory gets too low, never is trimmed in some caches, but not this one.

```suggestion
Pinned items with priority <xref:Microsoft.Extensions.Caching.Memory.CacheItemPriority.NeverRemove> are [never removed](https://github.com/dotnet/runtime/blob/release/6.0/src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs#L395-L427). The following code removes a cache item and calls `Compact` to remove 25% of cached entries:
```

Reply:
Yes, I think it's helpful. I'll commit the suggestion, but just focus the range slightly to cover the switch that decides what the priorities mean. Thanks!